boards/arm/frdm_k64f/pinmux.c

```diff
@@ -155,6 +155,15 @@ static int frdm_k64f_pinmux_init(const struct device *dev)
 		       PORT_PCR_PE_MASK | PORT_PCR_PS_MASK);
 #endif
 
+#if CONFIG_SHIELD_WAVESHARE_EPAPER_GDEH0213B72
```

Comment:
I do not think it is acceptable to insert snippets for all supported shields here.

Reply:
I see no other means for now. When eventually the pinmuxing is auto generated from the device tree this will all be past.
Alternatively all 'Arduino-compatible' boards should have a common default pinmuxing with these often used pins preconfigured for gpios